app/services/analytics_events.rb

```diff
@@ -633,6 +633,14 @@ def idv_in_person_location_visited(flow_path:, **extra)
     track_event('IdV: in person proofing location visited', flow_path: flow_path, **extra)
   end
 
+  def idv_in_person_location_searched(success:, errors: nil, **extra)
+    track_event('IdV: in person proofing location search submitted',
+     success: success,
+     errors: errors,
+     **extra,
+    )
```

Comment:
the indentation is off here, I'd also suggest bumping the string to the next line
```suggestion
    track_event(
      'IdV: in person proofing location search submitted',
      success: success,
      errors: errors,
      **extra,
    )
```